scss/_variables.scss

```diff
@@ -1967,26 +1967,29 @@ $back-to-top-icon-height:        subtract(1rem, 1px) !default;
 
 //// Stepped process
 // scss-docs-start stepped-process
-$stepped-process-max-items: 5 !default;
-$stepped-process-counter:   step !default; // Used as a counter name
-
-$step-item-margin-end:   $border-width !default;
-$step-item-padding:      .5rem !default;
-$step-item-bg:           $black !default;
-$step-item-active-bg:    $primary !default;
-$step-item-next-bg:      $gray-400 !default;
-$step-item-shadow-size:  $border-width * 1.5 !default;
-$step-item-drop-shadow:  drop-shadow($step-item-shadow-size 0 0 $white) #{"/* rtl:"} drop-shadow(-$step-item-shadow-size 0 0 $white) #{"*/"} !default;
-$step-item-padding-end:  $step-item-padding * 2 !default;
-
-$step-link-width:        1.25ch !default; // Matches width of a single number
-$step-link-color:        $white !default;
-$step-link-active-color: $black !default;
-$step-link-marker:       counter($stepped-process-counter) "\A0" !default;
-$step-link-marker-lg:    counter($stepped-process-counter) ".\A0" !default;
-
-$step-item-arrow-width:  1rem !default;
-$step-item-arrow-shape:  polygon(0% 0%, subtract(100%, $border-width) 50%, 0% 100%) #{"/* rtl:"} polygon(100% 0%, $border-width 50%, 100% 100%) #{"*/"} !default; // Used in clip-path
+$stepped-process-font-size:   $small-font-size !default;
+$stepped-process-font-weight: $font-weight-bold !default;
+$stepped-process-max-items:   5 !default;
+$stepped-process-counter:     step !default; // Used as a counter name
+
+$step-item-padding:       .5rem !default;
+$step-item-margin-end:    $border-width !default;
+$step-item-bg:            $black !default;
+$step-item-active-bg:     $primary !default;
+$step-item-next-bg:       $gray-400 !default;
+$step-item-shadow-size:   $border-width * 1.5 !default;
+$step-item-drop-shadow:   drop-shadow($step-item-shadow-size 0 0 $white) #{"/* rtl:"} drop-shadow(-$step-item-shadow-size 0 0 $white) #{"*/"} !default;
+// $step-item-padding-end:  $step-item-padding * 2 !default;
```

Comment:
If not used anymore we should use `// fusv-disable` and `// fusv-enable` to handle the deprecation of the var in v5.2.0 (and then remove it afterwards)